airflow/providers/amazon/aws/operators/quicksight.py

```diff
@@ -0,0 +1,100 @@
+# under the License.
+
+from typing import TYPE_CHECKING, Optional, Sequence
+
+from airflow.models import BaseOperator
+from airflow.providers.amazon.aws.hooks.quicksight import QuickSightHook
+
+if TYPE_CHECKING:
+    from airflow.utils.context import Context
+
+DEFAULT_CONN_ID = "aws_default"
+
+
+class QuickSightCreateIngestionOperator(BaseOperator):
+    """
+    Creates and starts a new SPICE ingestion for a dataset.
+    Also, helps to Refresh existing SPICE datasets
+
+    :param data_set_id:  ID of the dataset used in the ingestion.
+    :param ingestion_id: ID for the ingestion.
+    :param aws_account_id: Amazon Web Services account ID.
+    :param ingestion_type: Type of ingestion. Values Can be  INCREMENTAL_REFRESH or FULL_REFRESH.
+        Default FULL_REFRESH.
+    :param wait_for_completion: If wait is set to True, the time interval, in seconds,
+        that the operation waits to check the status of the QuickSight Ingestion.
+    :param check_interval: if wait is set to be true, this is the time interval
+        in seconds which the operator will check the status of the QuickSight Ingestion
+    :param aws_conn_id: The Airflow connection used for AWS credentials. (templated)
+         If this is None or empty then the default boto3 behaviour is used. If
+         running Airflow in a distributed manner and aws_conn_id is None or
+         empty, then the default boto3 configuration would be used (and must be
+         maintained on each worker node).
+    :param region: Which AWS region the connection should use. (templated)
+         If this is None or empty then the default boto3 behaviour is used.
+    """
+
+    template_fields: Sequence[str] = (
+        "data_set_id",
+        "ingestion_id",
+        "aws_account_id",
+        "ingestion_type",
+        "wait_for_completion",
+        "check_interval",
+        "aws_conn_id",
+        "region",
+    )
+    ui_color = "#ffd700"
+
+    def __init__(
+        self,
+        data_set_id: str,
+        ingestion_id: str,
+        aws_account_id: str,
+        ingestion_type: str = "FULL_REFRESH",
+        wait_for_completion: bool = True,
+        check_interval: int = 30,
+        aws_conn_id: str = DEFAULT_CONN_ID,
+        region: Optional[str] = None,
+        **kwargs,
+    ):
+        self.data_set_id = data_set_id
+        self.ingestion_id = ingestion_id
+        self.aws_account_id = aws_account_id
+        self.ingestion_type = ingestion_type
+        self.wait_for_completion = wait_for_completion
+        self.check_interval = check_interval
+        self.aws_conn_id = aws_conn_id
+        self.region = region
+        super().__init__(**kwargs)
+
+    def execute(self, context: "Context"):
+        hook = QuickSightHook(
+            aws_conn_id=self.aws_conn_id,
+            region_name=self.region,
+        )
+        self.log.info("Running the QuickSight Spice Ingestion on Dataset ID: %s)", self.data_set_id)
```

Comment:
```suggestion
        self.log.info("Running the Amazon QuickSight SPICE Ingestion on Dataset ID: %s)", self.data_set_id)
```

Based on other uses, it looks like this is an acronym.

Reply:
Done